varya/README.md

```diff
@@ -0,0 +1,127 @@
+
+## How to use it
+
+To use the system, simply duplicate the `vayra-child` theme directory and rename it `my-theme-name`. You’ll also want to do a search for `varia-child` strings and replace them with `my-theme-name` as follows:
+
+  - Search for: `'varya-child'` and replace with: `'my-theme-name'` (with quotes).
+  - Search for: `varya_child_` and replace with: `my_theme_name_`.
+  - Search for: `Text Domain: varia-child` and replace with: `Text Domain: my-theme-name` in _style.css_.
+  - Search for:  `varia-child` and replace with: `my_theme_name`.
+  - Search for: `varia-child-` and replace with: `my-theme-name-`.
+
+Soon this process will be replaced by a `theme-dev-util` (similar to [this](https://github.com/Automattic/theme-dev-utils)) that allows you to run a command and automatically produce a child theme with all the strings already replaced.
+
+## Simple Child Theme Structure 
+(See: `/varya-child`)
+
+Use this simple Varya child-theme example to familiarize yourself with the CSS-variables and to see how they influence the theme design when you change them. To start, open up the `/varya-child-simple/variables.css` file. Change any of the values, save the file, and refresh the frontend of your site to see the changes. To add more variables, view the [full list of available variables](https://github.com/Automattic/themes-workspace/blob/add/variatheme/varya/variables.css) from the Varya parent them and copy any variables you want in to your child theme. 
+The stylesheet structure for a child-theme cascades downward like this:
+
+### Frontend
+- `/varya/variables.css`
+- `/varya-child-simple/variables.css` (System Override)
+- `/varya/style.css`
+- `/varya-child-simple/style.css` (Extra CSS)
+
+### Editor
+- `/varya/variables-editor.css`
+- `/varya-child-simple/variables-editor.css` (System Overrides)
+- `/varya/style-editor.css`
+- `/varya-child-simple/style-editor.css` (Extra CSS)
+
+### System Overrides
+
+A list of CSS-variables that override the variables in the child theme. This is where you tell the system to use the _Futura_ font-family instead of the _sans-serif_ default, for example. There’s no need to replace all of the variables here, only the ones you wish to actually change based on your theme design.
+
+**Varya parent theme font style** (`/varya/variables.css`)
+
+`:root { --global--font-family: sans-serif; }`
+
+**Varya child-theme font-style overrides** (`/varya-child/variables.css`)
+
+`:root { --global--font-family: Futura, Helvetica; }`
+
+### Extra CSS
+
+These should be supplemental styles that give the theme a unique appearance beyond what’s possible with the Varya system. Need to add a fixed header or add a box-shadow to your theme’s buttons? This is where those styles would go. When possible and appropriate, try to include Varya CSS-variables in this styles sheet so that the system retains its usefulness across the theme.
+
+## Advanced Child Theme Structure 
+(See: `/varya-child-advanced`)
```

Comment:
I feel like this should eventually be a separate example with its own repo?

Reply:
Yeah, that's a good idea. But it's fine to keep this in here for now.

Reply:
Yeah, this is actually just the Varya version of “Nwaneri” so eventually it’ll be its own thing anyway.

Alternatively, I wonder if common code snippets might be useful to include here. With Varya themes you spend most of your time working on the header & footer UI since those areas are Gutenberg-able yet. 

If we were to come up with 2-3 common header patterns for example, we could create some Varya specific snippets that give you a jump start on styling those areas, at least until FSE is here.